src/Avalonia.Base/Input/Focus/FindNextElementOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Avalonia.Input
+{
+    /// <summary>
+    /// Provides options to help identify the next element that can programmatically
+    /// receive focus
+    /// </summary>
+    public class FindNextElementOptions
+    {
+        /// <summary>
+        /// Gets or sets the focus navigation strategy used to identify the best candidate
+        /// element to receive focus
+        /// </summary>
+        /// <remarks>
+        /// In UWP, a separate enum 'XYFocusNavigationStrategyOverride' exists, but holds same
+        /// values as XYFocusNavigationStrategy - so just using that one
+        /// </remarks>
+        public XYFocusNavigationStrategy XYFocusNavigationStrategyOverride { get; set; }
```

Comment:
This PR is going to be huge even without XYFocus. Wouldn't it be better to implement it in another PR?

Reply:
`FindNextElementOptions` is only included now so I could add the full FocusManager API. It isn't used until XYFocus is implemented, so it's more just a placeholder for now.